djorg/urls.py

```diff
@@ -14,14 +14,16 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path, include
+from django.urls import path, include, re_path
 from rest_framework import routers
 from notes.api import PersonalNoteViewSet
+from rest_framework.authtoken import views
 
 router = routers.DefaultRouter()
 router.register(r'notes', PersonalNoteViewSet)
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include(router.urls)),
+    re_path(r'^api-token-auth/', views.obtain_auth_token)
 ]
```

Comment:
Looks like you completed Day 4 correctly. Were you able to generate an auth token and test with cUrl?

Reply:
yup, tested with postman